app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -46,7 +46,7 @@ function DocumentsStep({
   return (
     <CaptureTroubleshooting>
       <PageHeading>{t('doc_auth.headings.document_capture')}</PageHeading>
-      {isMobile && <p>{t('doc_auth.info.document_capture_intro_acknowledgment')}</p>}
```

Comment:
I noticed this while making these changes--a paragraph of content was visible on mobile but not on desktop. I don't think there's any harm showing this language on desktop?